api/proto/auth/contextdialer.go

```diff
@@ -0,0 +1,44 @@
+package auth
+
+import (
+	"net"
+	time "time"
+
+	context "golang.org/x/net/context"
```

Comment:
Why do we need the old context package?
The stdlib `context` should work most of the time. If not, leave a comment explaining why